src/containers/CallLogs/CallLogs.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { getAllCalls } from '../../store/actions/callActions';
+import CallLog from '../../components/CallLog/CallLog.jsx';
+
+const mapStateToProps = state => {
+  return {
+    calls: state.callLogReducer.calls
+  };
+};
+
+class CallLogs extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    this.props.getAllCalls();
+  }
+
+  singleCallHandler(e) {
+    const callId = e.target.dataset.id;
+    this.props.history.push('/call/' + callId);
```

Comment:
also, tbh I think you can just use `Link` from `react-router-dom` to handle this logic. Doesn't need to be a function